lib/api/src/Cardano/Wallet/Api/Http/Server/Error/AssignReedemers.hs

```diff
@@ -72,129 +60,3 @@ instance IsRecentEra era => IsServerError (ErrAssignRedeemers era) where
                     , "into a valid Plutus data:"
                     , pretty r <> "."
                     ]
-        ErrAssignRedeemersTranslationError x -> case recentEra @era of
-            RecentEraBabbage -> fromBabbageContextError x
-            RecentEraConway -> fromConwayContextError x
-
-fromAlonzoContextError :: AlonzoContextError era -> ServerError
-fromAlonzoContextError = \case
-    TranslationLogicMissingInput inp ->
-        apiError err400 UnresolvedInputs
-            $ T.unwords
-                [ "The transaction I was given contains inputs I don't know"
-                , "about. Please ensure all foreign inputs are specified as "
-                , "part of the API request. The unknown input is:\n\n"
-                , T.pack $ show inp
-                ]
-    TimeTranslationPastHorizon t ->
-        apiError err400 PastHorizon
-            $ T.unwords
-                [ "The transaction's validity interval is past the horizon"
-                , "of safe slot-to-time conversions."
-                , "This may happen when I know about a future era"
-                , "which has not yet been confirmed on-chain. Try setting the"
-                , "bounds of the validity interval to be earlier.\n\n"
-                , "Here are the full details: " <> t
-                ]
-
-fromBabbageContextError
-    :: forall era
-     . IsRecentEra era
-    => BabbageContextError era
-    -> ServerError
-fromBabbageContextError = \case
-    AlonzoContextError e -> fromAlonzoContextError e
-    ByronTxOutInContext txOut ->
-        apiError err400 TranslationByronTxOutInContext
-            $ T.unwords
-                [ "The transaction I was given contains a Byron-style TxOut"
-                , "which is not supported when executing Plutus scripts. "
-                , "The offending TxOut is:\n\n"
-                , T.pack $ show txOut
-                ]
```

Comment:
This is not unreachable, but i'd question the worth of maintaining this error mapping now. I don't think anyone uses it.